Update readme (#1270)
microsoft/CCF: README.md

| | Link |
|- |- |
| Community Chat | [![Gitter](https://badges.gitter.im/MSRC-CCF/community.svg)](https://gitter.im/MSRC-CCF/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge) |
| Continuous Integration | [![Build Status](https://dev.azure.com/MSRC-CCF/CCF/_apis/build/status/CCF%20Github%20CI?branchName=master)](https://dev.azure.com/MSRC-CCF/CCF/_build/latest?definitionId=3&branchName=master) |

> | Documentation | [![docs](https://dev.azure.com/MSRC-CCF/CCF/_apis/build/status/CCF%20GitHub%20Pages?branchName=master)](https://microsoft.github.io/CCF/) |

Reviewer: Might be useful to maintain a badge link to our docs? Although it probably shouldn't be a build-status badge.

> [![Gitter](https://badges.gitter.im/MSRC-CCF/community.svg)](https://gitter.im/MSRC-CCF/community?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge) [![Build Status](https://dev.azure.com/MSRC-CCF/CCF/_apis/build/status/CCF%20GitHub%20Daily?branchName=master)](https://dev.azure.com/MSRC-CCF/CCF/_build/latest?definitionId=7&branchName=master)

Reviewer 2: If we keep only one build badge, it should be the CI, not the Daily.

The Confidential Consortium Framework (CCF) is an open-source framework for building a new category of secure, highly available,
and performant applications that focus on multi-party compute and data.
CCF can enable high-scale, confidential networks that meet key enterprise requirements

> — providing a means to accelerate production consortium based enterprise adoption of blockchain and multi-party compute technology.

Reviewer: I don't understand the "consortium based" added in the middle of this phrase.

```suggestion
— providing a means to accelerate production and enterprise adoption of blockchain and multi-party compute technology.
```

Leveraging the power of trusted execution environments (TEEs), decentralized systems concepts, and cryptography, CCF enables enterprise-ready multiparty computation or blockchains.

## Learn more and get started

 * Browse the [CCF Documentation](https://microsoft.github.io/CCF/).
 * Read the [CCF Technical Report](CCF-TECHNICAL-REPORT.pdf) for a more detailed description.
 * Learn more about [Azure Confidential Computing](https://azure.microsoft.com/solutions/confidential-compute/) offerings like Azure DC-series (which support Intel SGX TEE)
   and [Open Enclave](https://github.com/openenclave/openenclave) that CCF can run on.

## Getting Started with CCF

> * [Get started](https://microsoft.github.io/CCF/master/quickstart/index.html) with Azure confidentual compute and CCF. 

Reviewer: Not sure if we should be using "confidential compute" or "confidential computing" - [this](https://azure.microsoft.com/en-us/solutions/confidential-compute/) page uses one for the URL and another for the title. [These](https://docs.microsoft.com/en-us/azure/confidential-computing/overview) docs use "computing". Probably "ing"?

```suggestion
* [Get started](https://microsoft.github.io/CCF/master/quickstart/index.html) with Azure confidential computing and CCF. 
```

Reviewer 2: Something to confirm with Brent.

> * Learn how to [build](https://microsoft.github.io/CCF/master/quickstart/build.html) and [run](https://microsoft.github.io/CCF/master/quickstart/build.html#running-tests) a test network.

Reviewer 3: I think this bullet point should go last and only mention running a test network, pointing to https://microsoft.github.io/CCF/master/users/deploy_app.html. 

I don't think we should point at how to build CCF (or if we do, in a different bullet point) but point at the releases instead (https://github.com/microsoft/CCF/releases).

> * Start [building](https://microsoft.github.io/CCF/master/developers/index.html) your first CCF network.

Reviewer: We use build above. Clearer if we use a different term here.

```suggestion
* Start [writing](https://microsoft.github.io/CCF/master/developers/index.html) your own CCF app.
```
